packages/xo-web/src/xo-app/dashboard/health/unhealthyVdis.js

```diff
@@ -35,6 +35,11 @@ const COLUMNS = [
   {
     itemRenderer: (srId, { vdisHealthBySr }) => (
       <div>
+        <SingleLineRow>
+          <Col>
+            <em>{_('nVdis', { nVdis: size(vdisHealthBySr[srId].unhealthyVdis) })}</em>
```

Comment:
I believe this is incorrect, please take a look at this change: https://github.com/vatesfr/xen-orchestra/pull/7098/files#diff-9bbffcfb7f066db4478b3a52001fd94df3de3c951279bfdb10fe54d0f7877630

Reply:
I'm not sure but: here we're showing the list of VDIs that need to be coalesced. Not the length of the VDI chains. So we're only showing the `size` of the *collection*, we're not using the lengths that are stored in it.

Reply:
You are right, but I'm not sure this info is really relevant or it's what the user is asking.

Reply:
The user is saying that when there are more than 10 VDIs to coalesce, the warning's tooltip only says "More than 10" without saying how many exactly. So they're asking to show the explicit amount of VDIs. And the same value is used to decide whether to show the warning or not:

https://github.com/vatesfr/xen-orchestra/blob/1a1dd0531dbea87fdde25a30fec40db88a5082df/packages/xo-web/src/xo-app/dashboard/health/unhealthyVdis.js#L21-L30

So if this change is incorrect, then we'd need to fix the warning too.